entrypoint.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
+set -ex
 
+# install aws dependencies
+curl -o heptio-authenticator-aws https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-06-05/bin/linux/amd64/heptio-authenticator-aws
+chmod +x ./heptio-authenticator-aws
+mkdir -p $HOME/bin
+cp ./heptio-authenticator-aws $HOME/bin/heptio-authenticator-aws && export PATH=$HOME/bin:$PATH
+echo 'export PATH=$HOME/bin:$PATH' >> ~/.bashrc
```

Comment:
I suggest to install aws dependencies in a separate file, so if some fail occurred, exporting variables will work, what do you think?